File: defects-explorer/src/main/java/cn/edu/sustc/cse/sql/defectsexplorer/persistence/StackOverflowQARepository.java
package cn.edu.sustc.cse.sql.defectsexplorer.persistence;

import cn.edu.sustc.cse.sql.defectsexplorer.entities.StackOverflowQA;
import org.bson.types.ObjectId;
import org.springframework.data.mongodb.repository.MongoRepository;

public interface StackOverflowQARepository extends MongoRepository<StackOverflowQA, ObjectId> {
}

File: defects-explorer/src/main/kotlin/cn/edu/sustc/cse/sql/defectsexplorer/persistence/StackOverflowQARepository.kt
package cn.edu.sustc.cse.sql.defectsexplorer.persistence

import cn.edu.sustc.cse.sql.defectsexplorer.entities.StackOverflowQA
import org.springframework.data.mongodb.repository.MongoRepository

interface StackOverflowQARepository : MongoRepository<StackOverflowQA, Int>
